### PR TITLE
OpenShift Dev UI

### PR DESCRIPTION
**Pull request by cescoffier**
This commit provides the same UI as the previous Dev UI.

### CODE REVIEW

**Review by melloware — APPROVED** [review]
Nice work.

File: extensions/kubernetes/openshift/deployment/src/main/java/io/quarkus/openshift/deployment/OpenshiftDevConsoleProcessor.java [hunk 1]
```diff
@@ -13,4 +27,59 @@ DevConsoleRouteBuildItem builder() {
...
+    private Function<Map<String, String>, String> build() {
+        return (map -> {
+
+            QuarkusBootstrap existing = (QuarkusBootstrap) DevConsoleManager.getQuarkusBootstrap();
+            try (TempSystemProperties properties = new TempSystemProperties()) {
+                properties.set("quarkus.kubernetes.deploy", "true");
+                for (Map.Entry<String, String> arg : map.entrySet()) {
+                    properties.set(arg.getKey(), arg.getValue());
+                }
+                System.out.println("Running build..." + properties + " / " + map);
```
Comment (gastaldi) [inline extensions/kubernetes/openshift/deployment/src/main/java/io/quarkus/openshift/deployment/OpenshiftDevConsoleProcessor.java:61]: Are these `System.out` statements intended to stay?

**Review by iocanel — APPROVED** [review]
Looks great!

File: extensions/kubernetes/openshift/deployment/src/main/java/io/quarkus/openshift/deployment/OpenshiftDevConsoleProcessor.java (hunk 1 above)
Comment (cescoffier) [reply extensions/kubernetes/openshift/deployment/src/main/java/io/quarkus/openshift/deployment/OpenshiftDevConsoleProcessor.java:61]: Nope, that is a left-over from my long debugging session.

**Review by gastaldi — APPROVED** [review]

**Pull request merged**